docs/security/security.md

```diff
@@ -7,5 +7,5 @@ This section describes the methodology and means of securing a K3s cluster. It's
 
 The documents below apply to CIS Kubernetes Benchmark v1.23.
 
-* [Hardening Guide](hardening-guide.md)
-* [CIS Benchmark Self-Assessment Guide](self-assessment.md)
+* [Hardening Guide](./hardening-guide.md)
+* [CIS Benchmark Self-Assessment Guide](./self-assessment.md)
```

Comment:
Nit: you don't need this change, the links work just fine.

Reply:
Links updated